packages/maillog/graphql/schema-types.js

```diff
@@ -1,5 +1,9 @@
 module.exports = `
 
+input MaiLogFiltersInput {
+  errornous: Boolean
```

Comment:
typo, the thing is spelled: `erroneous`. Why not just `error`?

Reply:
`error` won't indicate a `Boolean` value, considering `MailLogRecord.error` is a `String`.

Settle for `hasError`!